src/exec/Main.hs

```diff
@@ -324,15 +322,22 @@ runShelleyServer
   -> IO ()
 runShelleyServer addr _ ctable rversions = do
   addrInfo <- resolveAddress addr
+  peerStatesVar <- newPeerStatesVar
   withServer
+    nullTracer
```

Comment:
I think that all these `nullTracers` will be problematic. It looks like there will be now visibility into any network issue on the Shelley side of the proxy.

Reply:
I don't disagree but they were already present, independent of this PR.

Reply:
Those `nullTracer`s can be addressed in a later PR.